Application/Source/Food Table/FoodTableViewModel.swift

```diff
@@ -0,0 +1,35 @@
+import RxSwift
+import RxCocoa
+import RxExtensions
+import Presentations
+import Action
+
+class FoodTableViewModel: ViewModel {
+
+    let isActive = BehaviorRelay(value: false)
+    let foods: BehaviorRelay<[Food]>
+
+    init(with foods: BehaviorRelay<[Food]>) {
+        self.foods = foods
+        foods
+            .asObservable()
+            .logValue(.info, .application) { "FOODS: \($0)" }
+            .subscribe()
+            .disposed(by: disposeBag)
+    }
+
+    private let disposeBag = DisposeBag()
+
+}
+
+protocol FoodTableViewModelFactoryProtocol {
+    func makeFoodTableViewModel(with foods: BehaviorRelay<[Food]>) -> FoodTableViewModel
```

Comment:
`foods` should be a property of `FoodTableViewModelFactoryProtocol`—not passed in—since it is a dependency of `FoodTableViewModel`.

We should also make `foods` a property of the `DetailViewModelFactoryProtocol`, and then both would inject `foods` into their corresponding view models when they are created.